prisma/schema.prisma

```diff
@@ -27,6 +27,7 @@ model Ingredient {
   img           String
   nutritionInfo String
   recipes       IngredientsOnRecipes[]
+  price         Price?
```

Comment:
You want to have a 1-to-many relationship so you can keep track of price changes over an extended period of time.